src/github.com/getlantern/flashlight/proxied/proxied_test.go

```diff
@@ -0,0 +1,163 @@
+package proxied
```

Comment:
This is just based on the old util_test, minus the hash stuff which is still in the util package.